docs/contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contribute to SQLite for Deno
+
+?> This is a draft.
+
+Thank you for considering to contribute to the SQLite for Deno module! Below are a few guidelines
+on how to contribute.
+
+## Prerequisites
+
+To work on the JavaScript/ TypeScript wrapper module, all you need is a [deno](https://deno.land)
+runtime.
+
+To change the compiled SQLite WASM binary, you will require both the [WASI SDK][wasi-sdk] and the
```

Comment:
(Now I kind of understand why my compilation didn't work correctly -- I used the clang compiler from `emscripten` and not the full `WASI-SDK` suite. :sweat_smile:)